cns/configuration/configuration.go

```diff
@@ -36,7 +36,7 @@ type CNSConfig struct {
 	EnableIPAMv2                bool
 	EnablePprof                 bool
 	EnableSubnetScarcity        bool
-	EnableSwiftV2               bool
```

Comment:
Double check if we are still referencing this elsewhere.

Reply:
checked by doing a search project-wide, we don't use it anywhere else